Makefile

```diff
@@ -6,4 +6,6 @@ image:
 	docker build --rm -t $(IMAGE_NAME) .
 
 test: image
-	docker run --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && bundle exec rake"
+	docker run --rm \
+		--workdir /usr/src/app
+		$(IMAGE_NAME) CODECLIMATE_REPO_TOKEN=ca5120f90202a99c43c0914d301a94e5908136d146ecc0d7fe7fde1c3f9d325e bundle exec rake
```

Comment:
ah, I see, that seems preferable. Running the test suit locally via `make test` doesn't need to report to CC unless you explicitly ask it to. I'll make that change.